dev/SapphireTest.php

```diff
@@ -496,10 +496,6 @@ public function tearDown() {
 		// Preserve memory settings
 		ini_set('memory_limit', ($this->originalMemoryLimit) ? $this->originalMemoryLimit : -1);
 
-		// Restore email configuration
```

Comment:
Nope - when I was debugging this $mailer was set to an instance of TestMailer it was just somehow a different instance to what was in Injector. I had assumed they would reference the same object but they weren't and I couldn't figure out why.